trunk/Lib/Date.php

```diff
@@ -18,33 +18,39 @@ class Date {
     /**
      * Filter date-output from date_i18n() to return always a timestamp
      *
-     * @param string    $j             Formatted date string.
-     * @param string    $req_format    Format to display the date.
-     * @param int       $i             Unix timestamp.
-     * @param bool      $gmt           Whether to convert to GMT for time. Default false.
+     * @param string $j Formatted date string.
+     * @param string $reqFormat Format to display the date.
+     * @param int $i Unix timestamp.
+     * @param bool $gmt Whether to convert to GMT for time. Default false.
      * @return int Unix timestamp
      */
-    public function filterDateI18n( $j, $req_format, $i, $gmt ) {
+    public function filterDateI18n( $j, $reqFormat, $i, $gmt ) {
```

Comment:
I think it would be more readable if these Parameter names would be better named  like

```suggestion
    public function filterDateI18n( $formattedDate, $reqFormat, $unixTimestamp, $gmt  ) {
```

Reply:
done